aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -1017,12 +1027,14 @@ Tensor cudnn_convolution_backward_input(
 {
   checkAllSameType(c, {grad_output, weight});
   checkAllSameGPU(c, {grad_output, weight});
-
   auto layout = support_channels_last(*grad_output, *weight, groups) ?
       at::MemoryFormat::ChannelsLast : at::MemoryFormat::Contiguous;
   auto grad_input_t = at::empty(input_size, grad_output->options(), layout);
 
   // Avoid "grad_input" when this is being used as transposed convolution
+  if (grad_input_t.numel() == 0) {
+    return grad_input_t;
```

Comment:
I think you have to `empty_like` here, it's not OK to return the input tensor directly.